ci-operator/step-registry/ipi/conf/ipi-conf-ref.yaml

```diff
@@ -1,6 +1,9 @@
 ref:
   as: ipi-conf
-  from: base
+  from_image:
+    namespace: origin
+    name: centos
+    tag: '8'
```

Comment:
Why this change?  `base` seems less opinionated for steps that just need generic POSIX stuff, and it's one less thing to forget to bump.